docs/userguides/legalhold.md

```diff
@@ -0,0 +1,96 @@
+# Manage legal hold custodians
+
+Once you [create a legal hold matter in the Code42 console](https://support.code42.com/Administrator/Cloud/Configuring/Create_a_legal_hold_matter#Step_1:_Create_a_matter), you can use the Code42 CLI to add or release custodians from the matter.
+
+Use the `legal-hold` commands to manage legal hold custodians.
+ - To see a list of all the users currently in your organization, you can export a list from the [Users action menu](https://support.code42.com/Administrator/Cloud/Code42_console_reference/Users_reference#Action_menu).
+ - To view a list of legal hold matters for your organization, including the matter ID, use the following command:
+   `code42 legal-hold list`
+ - To see a list of all the custodians currently associated with a legal hold matter, enter `code42 legal-hold show <matterID>`.
+
+
+## Get CSV template
+
+To add multiple custodians to a legal hold matter:
+
+1. Generate a CSV template. Below is an example command for generating a template to use to add custodians to legal hold matter. Once generated, the CSV file is saved to your current working directory.
+    `code42 legal-hold bulk generate-template add`
+
+    To generate a template to use to remove custodians from a legal hold matter, for example:
```

Comment:
same here, "to use" seems out of place, at least from readability perspective, imo.  Keep it as is if others feel it is fine.

Few suggestions which are closer to intended meaning:

To generate a template and then use it to remove custodians...
To generate a template and use it to remove....
To generate a template then use it to remove...
To generate a template to remove custodians....

Reply:
To generate a template that can be used to remove...